server/controllers/appointments/deleteAppointments.js

```diff
@@ -0,0 +1,18 @@
+const { deleteAppointmentsQueries } = require('../../database/queries');
+
+const deleteAppointments = async (req, res, next) => {
+  try {
+    const { patientId, appointmentId } = req.params;
```

Comment:
Yuo need to validate the `appointmentId`
```suggestion
    const { appointmentId } = req.params;
```